homeassistant/components/mqtt/binary_sensor.py

```diff
@@ -43,12 +49,15 @@
 DEFAULT_PAYLOAD_OFF = "OFF"
 DEFAULT_PAYLOAD_ON = "ON"
 DEFAULT_FORCE_UPDATE = False
+CONF_AVAILABILITY_TOPIC = "availability_topic"
```

Comment:
This is defined in mqtt/__init__.py and should not be redefined here:
https://github.com/home-assistant/home-assistant/blob/6acfede512dcb2c0f41b5d591a3a2794e4299192/homeassistant/components/mqtt/__init__.py#L80